test/models/drive_test.dart

```diff
@@ -0,0 +1,1175 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/drives/models/drive.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+import 'package:motis_mitfahr_app/util/search/position.dart';
+
+import '../util/factories/drive_factory.dart';
+import '../util/factories/model_factory.dart';
+import '../util/factories/profile_factory.dart';
+import '../util/factories/ride_factory.dart';
+import '../util/mock_server.dart';
+import '../util/mock_server.mocks.dart';
+
+// Die Klasse UrlProcessor muss zu Beginn jeder Testdatei implementiert werden und die Methode processUrl überschrieben werden
+// Wird die Methode ProcessUrl aufgrufen, wird für den dort definierten Fall (in dem Beispiel client.from('drives').select('driver_id,seats')) die Antwort definiert
+// Die Datenbankabfrage an den Client wird so abgefangen und das gewünschte Ergebnis zurückgegeben.
+// Um herauszufinden welche URL durch die jeweilige Datenbankabfrage generiert wird, einfach den auskommentierten Print-Aufruf in der mockServer.Dart Datei aktivieren
+
+void main() {
+  final MockUrlProcessor driveProcessor = MockUrlProcessor();
+  //setup muss in jeder Testklasse einmal aufgerufen werden
+  setUp(() async {
+    MockServer.setProcessor(driveProcessor);
+  });
+  group('create a Drive', () {
+    test('The Types are correct for a Drive', () {
```

Comment:
I don't think that's a useful test, how could those types be wrong?

Reply:
ok